contents/epel.mdx

```diff
@@ -22,11 +22,18 @@ EPEL(Extra Packages for Enterprise Linux) 是由 Fedora Special Interest Group 
 <CodeBlock>
 
 ```shell
+# 对于 CentOS 7
 {{sudo}}sed -e 's!^metalink=!#metalink=!g' \
     -e 's!^#baseurl=!baseurl=!g' \
     -e 's!http://download\.fedoraproject\.org/pub/epel!{{http_protocol}}{{mirror}}!g' \
     -e 's!http://download\.example/pub/epel!{{http_protocol}}{{mirror}}!g' \
     -i /etc/yum.repos.d/epel*.repo
+
+# 对于 CentOS Stream 9
+{{sudo}}sed -e 's!^metalink=!#metalink=!g' \
+    -e 's!^#baseurl=!baseurl=!g' \
+    -e 's!https://download\.example/pub/epel!{{http_protocol}}{{mirror}}!g' \
```

Comment:
This is nearly the same command as above. No need to add them.

The extra `-e 's!http://download\.fedoraproject\.org/pub/epel!{{http_protocol}}{{mirror}}!g' \`, when applied on centos stream 9, should not have any effect I assume.

Reply:
The modification is primarily attributed to a minor issue. 

Specifically, the baseurl line in CentOS Stream was altered from using HTTP to utilizing HTTPS. 

That is:
In CentOS 7

```bash
#baseurl=http://
```
in CentOS Stream 9
```bash
#baseurl=https://
```
To provide suggestions without including actual code, it can be considered to rephrase help as follows?

"The adjustment in CentOS Stream from a minor inconvenience, where the baseurl line was transitioned from HTTP to HTTPS. For CentOS 7, the baseurl line initially resembled the following: ```#baseurl=http://```. However, in CentOS Stream 9, the baseurl line was updated to incorporate HTTPS with the entry: ```#baseurl=https://```."

Reply:
那可以把原来的变成 `https\?`

例如 
https://github.com/mirrorz-org/mirrorz-help/blob/543b9ebbc16c7537be1064b4741f262b22203feb/contents/alpine.mdx?plain=1#L15

Reply:
这似乎是一个不错的选择

忘记可以修改正则语法了